docs/reference/data-type-support.rst

```diff
@@ -0,0 +1,805 @@
+
+        * 
+          - :ref:`SYMM <hipblas_symm>`
+          - Symmetric matrix-matrix multiplication: :math:`C = \alpha A B + \beta C`
+          - ✅
+          - ✅
+
+        * 
+          - :ref:`SYRK <hipblas_syrk>`, :ref:`SYR2K <hipblas_syr2k>`
+          - Update symmetric matrix with one matrix product or by using two matrices.
+          - ✅
+          - ✅
+
+        * 
+          - :ref:`SYRKX <hipblas_syrkx>`
+          - SYRKX adds an extra matrix multiplication step before updating the symmetric matrix.
+          - ✅
+          - ✅
+
+        * 
+          - :ref:`TRMM <hipblas_trmm>`
+          - Triangular matrix-matrix multiplication.
+          - ✅
+          - ✅
+
+        * 
+          - :ref:`TRSM <hipblas_trsm>`
+          - Triangular solve with multiple right-hand sides.
+          - ✅
+          - ✅
+        * 
+          - :ref:`HEMM <hipblas_hemm>`
+          - Hermitian matrix-matrix multiplication.
+          - ✅
+          - ✅
+
+        * 
+          - :ref:`HERK <hipblas_herk>`, :ref:`HER2K <hipblas_her2k>`
+          - Update Hermitian matrix with one matrix product or by using two matrices.
+          - ✅
+          - ✅
+
+        * 
+          - :ref:`HERKX <hipblas_herkx>`
+          - HERKX adds an extra matrix multiplication step before updating the Hermitian matrix.
+          - ✅
+          - ✅
+
+        * 
+          - :ref:`TRTRI <hipblas_trtri>`
+          - Triangular matrix inversion.
+          - ❌
+          - ❌
+
+        * 
+          - :ref:`DGMM <hipblas_dgmm>`
+          - Diagonal matrix matrix multiplication.
+          - ✅
+          - ✅
+
+
+Extensions
+----------
+
+The :ref:`extension functions <hipblas_extension>` data type support is listed
+separately for every function for the different backends in the
+:ref:`rocBLAS extensions <rocblas:extension>` and
+`cuBLAS extensions <https://docs.nvidia.com/cuda/cublas/index.html#blas-like-extension>`_ 
+documentation.
+
+SOLVER API
+----------
+
+:ref:`Solver API <solver_api>` is for solving linear systems, computing matrix
+inverses, and performing matrix factorizations.
+
+.. tab-set::
+
+  .. tab-item:: Float types
+    :sync: float-type
+
+    .. list-table::
+        :header-rows: 1
+        :widths: 15, 45, 10, 10, 10, 10
+
+        * 
+          - Function
+          - Description
+          - float16
+          - bfloat16
+          - float
+          - double
+
+        * 
+          - :ref:`GETRF <hipblas_getrf>`
+          - Compute the LU factorization of a general matrix using partial pivoting with row interchanges.
+          - ❌
+          - ❌
+          - ✅
+          - ✅
+
+        * 
+          - :ref:`GETRS <hipblas_getrs>`
+          - Solve a system of linear equations :math:`AxX=B` after performing an LU factorization using GETRF.
+          - ❌
+          - ❌
+          - ✅
+          - ✅
+
+        * 
+          - :ref:`GETRI <hipblas_getri>`
+          - Compute the inverse of a matrix using its LU factorization.
```

Comment:
Only the batched version of getri is supported, not sure if that's noted somewhere.

Reply:
Fixed.